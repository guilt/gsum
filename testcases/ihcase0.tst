Name: ihcase0
Description: Test 20M tiled file with 1M chunks, incremental 5% hashing
File: test.txt 20M tile 1M 0x00,0xff
AlgoMatrix: default, bcrypt-sha512 12341234, crc32
Increment: 5%
