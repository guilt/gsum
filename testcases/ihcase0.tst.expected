Hash-default-0%-5%: 30e14955ebf1352266dc2ff8067e68104607e750abb9d3b36582b8af909fcb58
Hash-bcrypt-sha512-0%-5%: $2a$10$Io0q6yskeDLtSVUyJ/dHn.D7OWLcrRcxZDFdAmpGK8LvtK6D4YVbO
Hash-crc32-0%-5%: a738ea1c
Hash-default-5%-10%: f5fb04aa5b882706b9309e885f19477261336ef76a150c3b4d3489dfac3953ec
Hash-bcrypt-sha512-5%-10%: $2a$10$IWIhdspqNqAa8z71YsE7K.506AicST4VIwvP2NYCgvU.OE1RQ296C
Hash-crc32-5%-10%: 956bac74
